README.md

```diff
@@ -57,11 +56,16 @@ instance:
       --set gitea.config.server.ROOT_URL=http://gitea-http.${NAMESPACE}:3000/
   ```
 
-* Keptn must be configured to use the keptn-gitea-provisioner-service to automatically provision git repositories:
+* Keptn must be configured to use the keptn-gitea-provisioner-service to automatically provision git repositories. The flag is different for Keptn version 0.16 and version 0.17 and onwards:
   ```bash
   #!/bin/bash
   NAMESPACE=default
   
+  # Keptn 0.17
+  helm upgrade -n keptn keptn keptn/keptn \
+    --set "control-plane.features.automaticProvisioning.serviceURL=http://keptn-gitea-provisioner-service.${NAMESPACE}"
```

Comment:
You have to remove `control-plane.` from this

Reply:
Has been removed!